etc/adminhtml/events.xml

```diff
@@ -9,4 +9,7 @@
     name="admin_system_config_changed_section_klaviyo_reclaim_user">
         <observer name="custom_admin_system_config_changed_section_klaviyo_reclaim_user" instance="Klaviyo\Reclaim\Observer\KlaviyoUserObserver"/>
     </event>
+    <event name="catalog_product_delete_after">
```

Comment:
It would be nice to document that the above is a standard thing that gets called in magento2 for when a product is deleted https://magento.stackexchange.com/questions/300931/magento-2-delete-event-get-product-skus-using-observer

Reply:
this is a standard observer in magento, i think this is pretty self explanatory ?